common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4073,6 +4075,22 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(3)
 
+  val WORKER_WRITER_CREATE_INDEX_OR_SUCCESS_MAX_ATTEMPTS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.writer.create.indexOrSuccess.maxAttempts")
```

Comment:
```suggestion
    buildConf("celeborn.worker.writer.create.file.maxAttempts")
```
It's better to use `file` instead of `indexOrSuccess`.